packages/backend/src/payment-method/ilp/spsp/middleware.ts

```diff
@@ -41,11 +38,11 @@ const spspMiddleware = async (
   if (ctx.accepts('application/spsp4+json')) {
     const walletAddressService = await ctx.container.use('walletAddressService')
 
-    const walletAddress = await walletAddressService.getByUrl(
+    const walletAddress = await walletAddressService.getOrPollByUrl(
       ctx.walletAddressUrl
     )
 
-    if (!walletAddress?.isActive) {
```

Comment:
already handled in `getOrPollByUrl` by default. However, this comment still stands: https://github.com/interledger/rafiki/pull/2708/files#r1592829255